docs/production/authentication/microsoft.mdx

```diff
@@ -181,6 +181,24 @@ You may need to restart your Mage server for the changes to take effect.
 
 </AccordionGroup>
 
+### Configurable environment variables
+To configure the environment for Microsoft Azure Cloud and Microsoft Azure Gov Cloud sign-in, 
+you need to set the following environment variables:
+1. You can specify the `AZURE_HOST` URLs based on your environment:
+```
+AZURE_PUBLIC_CLOUD_HOST_URL="https://login.microsoftonline.com/"
+AZURE_CHINA_HOST_URL="https://login.chinacloudapi.cn/"
+AZURE_GERMANY_HOST_URL="https://login.microsoftonline.de/"
+AZURE_GOVERNMENT_HOST_URL="https://login.microsoftonline.us/"
+```
+2. You can specify the `MICROSOFT_GRAPH_API_HOST` URLs based you your environment:
```

Comment:
Reword this section similar to [above](https://github.com/mage-ai/mage-ai/pull/5352/files#r1727738429)

Reply:
May need to update values based on this [doc](https://learn.microsoft.com/en-us/graph/deployments#microsoft-graph-and-graph-explorer-service-root-endpoints)

Reply:
make sure to indicate what each url is used for (e.g. `Global service: "https://graph.microsoft.com")